apps/extension/src/shared/containers/word-length-toogles.tsx

```diff
@@ -0,0 +1,28 @@
+import { SeedPhraseLength } from 'penumbra-crypto-ts/src/mnemonic';
+import { Toggle } from 'ui/components';
+
+interface WordLengthTooglesProsp {
+  toogleClick: (length: SeedPhraseLength) => void;
+  phrase: string[];
+}
+
+export const WordLengthToogles = ({ toogleClick, phrase }: WordLengthTooglesProsp) => {
```

Comment:
If the toggle is selected, it should be white text. In this case `12 words` should be white.

<img width="313" alt="Screenshot 2023-09-20 at 2 38 05 PM" src="https://github.com/penumbra-zone/web/assets/16624263/2c9d42a2-12cf-4940-bdaf-b698505e4230">